cmd/influx_stress/example.toml

```diff
@@ -0,0 +1,82 @@
+[write]
+  concurrency = 10
+  batch_size = 5000
+  batch_interval = "0s"
+  database = "stress"
+  precision = "n"
+  address = "localhost:8086"
+  reset_database = true
+  # doesnt do anything
+  starting_point = "6m" # how far back in time to go
+
+[[series]]
+  point_count = 10 # number of points that will be written for each of the series
+  measurement = "cpu"
+  series_count = 100000
```

Comment:
`series_count` specifies how many series will be created for the specified `[[series]]`. For example, a series with `series_count=5` will create the following series:

`cpu,host=idk-0,location=lame-0 value=100`
`cpu,host=idk-1,location=lame-1 value=100`
`cpu,host=idk-2,location=lame-2 value=100`
`cpu,host=idk-3,location=lame-3 value=100`
`cpu,host=idk-4,location=lame-4 value=100`

I'm not a huge fan of this, but it seemed good enough for now.